app/jobs/remote_server_job.rb

```diff
@@ -13,59 +26,39 @@ def perform(initiator_key, benchmark, options = {})
       password: secrets.bare_metal_server_password
     ) do |ssh|
 
-      send(benchmark, ssh, initiator_key, options)
+      send(benchmark_group, ssh, initiator_key, options)
```

Comment:
@noahgibbs What do you think about calling method like this? Is it a good approach?